ui/src/app/pages/data/data-page.tsx

```diff
@@ -110,7 +110,8 @@ export const DataPage = withCurrentWorkspace()(class extends React.Component<
       this.setState({
         isLoading: true
       });
-      const results = await workspacesApi().getCdrSelectors(namespace, id);
+      const results = await workspacesApi().getWorkspaceResources(namespace, id,
+        [ResourceType.COHORT, ResourceType.COHORTREVIEW, ResourceType.CONCEPTSET, ResourceType.DATASET]);
```

Comment:
I'd likely define this array someplace handy and call it `DATA_PAGE_WORKSPACE_RESORCE_TYPES` or some such, even if there's only one use. (My issue with the 2- or 3-use rule is that the next person may not bother to count the uses, and then it's like a riptide.)